### PR TITLE
fix(ingest): lower nb of documents to ingest

### PR DESCRIPTION
**Pull request by lionelB**

### CODE REVIEW

File: targets/ingester/src/cli.js [hunk 1]
```diff
@@ -160,7 +186,8 @@ async function insertDocuments(docs) {
     .toPromise();
 
   if (result.error) {
-    console.error(result.error);
+    console.log(docs.map(({ id }) => id));
```
Comment (UnbearableBear) [inline targets/ingester/src/cli.js:189]: il est voulu ce log ?

**Review by UnbearableBear — APPROVED** [review]
👍

File: targets/ingester/src/cli.js (hunk 1 above)
Comment (lionelB) [reply targets/ingester/src/cli.js:189]: no no no =)

Comment (lionelB) [inline targets/ingester/src/cli.js:189]:
```suggestion 
```

**Pull request merged**